services/tasks.js

```diff
@@ -83,7 +58,31 @@ const addTaskCreatedAtAndUpdatedAtFields = async () => {
   return operationStats;
 };
 
+const fetchOrphanedTasks = async () => {
+  try {
+    const userSnapshot = await usersQuery.fetchUsersNotInDiscordServer();
+    if (userSnapshot.empty) {
+      return [];
+    }
+    const userIds = userSnapshot.docs.map((doc) => doc.id);
+
+    const abandonedTasks = [];
```

Comment:
remove the line between them.

Reply:
Is this necessary?
I feel it looks neat and readable.

Reply:
We should group variable declaration and add the blank lines before and after the block like for and if.
What do you think?

Reply:
I don't have a strong preference for a particular style. Do we follow any specific coding guidelines or conventions for spacing in the project?